clients/drcachesim/common/options.cpp

```diff
@@ -374,6 +374,18 @@ droption_t<std::string> op_raw_compress(
     "for an SSD, zlib and gzip typically add overhead and would only be used if space is "
     "at a premium; snappy_nocrc and lz4 are nearly always performance wins.");
 
+droption_t<std::string> op_trace_compress(
+    DROPTION_SCOPE_FRONTEND, "compress", DEFAULT_TRACE_COMPRESSION_TYPE,
+    "Trace compression: \"gzip\",\"zlib\",\"lz4\",\"none\"",
```

Comment:
s/zlib/zip/.  (Best to put zip first too to match the list on the line below and b/c it is the expected format for full feature support.)